### ISSUE TITLE
Unable to resize item sheets/unable to use 'legacy' item sheets

### ISSUE THREAD

**Issue by Pbmarsh** [foundryvtt/dnd5e#4239]
Hi, I'm combining these as one issue ticket, as either one would fix the issue in question, but please let me know if they should be separated. 

Since the 4.0 release of the DnD5e system, I am no longer able to resize the new item sheets. My first thought was to try and roll the sheets back to the 'legacy' version, like you can with character sheets. 

Beyond this just being an annoyance on it's own, the main direct impact this is having for me right now is that if I have too many modules enabled which add functionality to items (such as all of the modules I had activated before), the top of the item sheet just completely disappears. 
In this first screen shot I have 3 modules activated that add buttons to the top of the sheet, and as you can see this is getting cut off, but not completely.
![image](https://github.com/user-attachments/assets/0edd859d-5844-4309-806a-578541a48e41)
In this screenshot I have activated an additional module that adds a button, so 3 seems to be the (rough) limit - that said I suspect things like the title of the item, source of the item, and whether it's a magical item with a magical rune seem to also impact it. 
![image](https://github.com/user-attachments/assets/5ac0bf10-23c6-4959-9805-10088e92686d)
My reason for believing this is here's a screenshot of the arrows sheet as a point of comparison, with the same number of modules activated as in the first screenshot: 
![image](https://github.com/user-attachments/assets/6cce086b-d4ca-490d-92e5-62cf39ddaaee)

As a result, it seems I'm now limited by how many modules I can have activated at once - while this could be fixed by raising issues with the individual module developers, I personally believe that even if this wasn't affecting the modules, the inability to resize the window would still be an annoying limitation, and fixing it on the system side of things would inherently solve the module side of things.

**Comment by JPMeehan** [comment]
The no-legacy issue is something that's an intentional change, per the patch notes
> The legacy sheets are no longer available, as making them available would have required us to duplicate all the additional work done to the V2 sheets in order to support Activities.

I agree that the system should add resizability to the sheets, at least until the eventual AppV2 refactor shoves all those window buttons into a dropdown.

**Comment by Pbmarsh** [comment]
> The no-legacy issue is something that's an intentional change, per the patch notes
> 
> > The legacy sheets are no longer available, as making them available would have required us to duplicate all the additional work done to the V2 sheets in order to support Activities.
> 
> I agree that the system should add resizability to the sheets, at least until the eventual AppV2 refactor shoves all those window buttons into a dropdown.

Noted - I have changed the title

**Comment by M0nk3yy** [comment]
I think overflow handling is the issue here rather than the need for resizable sheets. If a full header was spread onto a second line for example, that would provide a usable experience.

I think it is worth noting that the biggest space hog in your example is your Source field, not the number of modules. Generally these are listed using an abbreviations (SRD, TCoE, etc.). For example I can use 3 sheet header mods here without issue:

![image](https://github.com/user-attachments/assets/23b04eb8-5895-4fbb-850a-4887e1b7c05f)

But adding a single extra X causes the overflow because it has become too lengthy.

Regardless, obviously this is an issue that needs better handling, especially when the buttons disappear entirely.

**Comment by JPMeehan** [comment]
Yeah, I don't really want to edit hundreds of items because the spacing standards on the source field changed — I ran into this with zero modules on my copy of the Beastheart class with the source "Beastheart and Monstrous Companions"

**Comment by Pbmarsh** [comment]
> I think overflow handling is the issue here rather than the need for resizable sheets. If a full header was spread onto a second line for example, that would provide a usable experience.
> 
> I think it is worth noting that the biggest space hog in your example is your Source field, not the number of modules. Generally these are listed using an abbreviations (SRD, TCoE, etc.). For example I can use 3 sheet header mods here without issue:
> 
> ![image](https://private-user-images.githubusercontent.com/66137312/367583146-23b04eb8-5895-4fbb-850a-4887e1b7c05f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjY0Mjc5MjEsIm5iZiI6MTcyNjQyNzYyMSwicGF0aCI6Ii82NjEzNzMxMi8zNjc1ODMxNDYtMjNiMDRlYjgtNTg5NS00ZmJiLTg1MGEtNDg4N2UxYjdjMDVmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTE1VDE5MTM0MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTBkNThlOTc4NDcxM2FmNDY4MjRiOTc5MmY0ZTgxY2E0OTYwZmU1NzYyYjgzMzlhMDJjOWRhNGZmNGU4NzExMTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.DF8MI22MbB8CpRDFWX4jGXEVmgOIMXYQskbxpHQuUlw)
> 
> But adding a single extra X causes the overflow because it has become too lengthy.
> 
> Regardless, obviously this is an issue that needs better handling, especially when the buttons disappear entirely.

Yeah I came across this same work around last night - by removing the source from several items I tested with, I was able to get all of my module buttons to appear. But as JPMeehan has said, doing that for each item will be a massive pain - hopefully it's something that can be fixed in some way for the next update!

**Comment by krbz999** [comment]
A long label like "Icewind Dale: Rime of the Frostmaiden" will make the buttons completely inaccessible. WotC makes long book names. This is likely affecting a lot of people at the moment.

**Comment by JPMeehan** [comment]
Especially since there weren't any conventions or standards for what the Source: Book field meant or how it was used really prior to this update.
